lib/lotus/model/adapters/sql_adapter.rb

```diff
@@ -281,6 +281,28 @@ def disconnect
 
         private
 
+        # Returns a hash of attributes and their default values
+        #
+        # @api private
+        # @since x.x.x
+        def defaults(collection)
```

Comment:
It looks like #schema is actually on the Sequel connection object not the Sequel dataset object